index.html

```diff
@@ -639,19 +639,17 @@ <h2>Core Data Model</h2>
       </p>
 
       <p class="note"
-        title="Abstract concepts vs. concrete concepts">
-Readers might note that some abstract concepts described in this section
-do not have normative requirements nor associated media types defined by this
-specification. For example, the abstract concepts of <a>credential</a> and
-<a>presentation</a> do not have normative requirements nor associated media
-types. However, the concrete concepts of a <a>verifiable credential</a> or a
-<a>verifiable presentation</a> are defined as <a>conforming documents</a> and
-do have associated media types. Furthermore, the only difference between the
-abstract concepts — of <a>credential</a> and <a>presentation</a> — and the
-concrete concepts — of <a>verifiable credential</a> and
-<a>verifiable presentation</a> — is the fact that the "verifiable" objects
-are secured in a way that is cryptographically verifiable. For more details,
-see Section <a href="#securing-verifiable-credentials"></a>.
+        title="The difference between a credential and a verifiable credential">
+Readers might note that some concepts described in this section, such as
+<a>credential</a> and <a>presentation</a>, do not have media types defined by
+this specification. However, the concepts of a <a>verifiable credential</a> or a
+<a>verifiable presentation</a> are defined as <a>conforming documents</a> and do
+have associated media types. The concrete difference between the
+concepts &mdash; of <a>credential</a> and <a>presentation</a> &mdash; and the
+concepts &mdash; of <a>verifiable credential</a> and <a>verifiable
+presentation</a> &mdash; is the fact that the "verifiable" objects are secured
+in a way that is cryptographically verifiable. For more details, see Section
+<a href="#securing-verifiable-credentials"></a>.
```

Comment:
Further massage may be needed. This is better than what it would replace, anyway.
```suggestion
have associated media types. The concrete difference between these concepts
&mdash; between <a>credential</a> and <a>presentation</a> Vs. <a>verifiable
credential</a> and <a>verifiable presentation</a> &mdash; is simply the fact
that the "verifiable" objects are secured in a way that is cryptographically
verifiable, which the others are not. For more details, see Section
<a href="#securing-verifiable-credentials"></a>.
```